runtimes/bundles/org.jboss.tools.rsp.server.wildfly/src/main/java/org/jboss/tools/rsp/server/wildfly/runtimes/download/DownloadManagerWorkflowUtility.java

```diff
@@ -55,7 +60,8 @@ public static int getWorkflowStatus(DownloadRuntime dr, String userS, String pas
 		if (response == 401) {
 			// 401 means bad credentials, change nothing
 			return CREDENTIALS_FAILED;
-		} else if (response == 403 || response == 200) {
+		} else if (response == 403
+				|| response == 200) {
```

Comment:
Why does it move this to a new line? It's not even very long :(   I hate automatic code formatters :(

Reply:
this, too. On purpose. Like you prefer not to immediately return a value but to store in a temporary variable and only return afterwards: I prefer to have every condition on it's own line and get better readable imho.